taxi_service/settings.py

```diff
@@ -20,9 +22,8 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-8ovil3xu6=eaoqd#-#&ricv159p0pypoh5_lgm*)-dfcjqe=yc"
-)
+SECRET_KEY = ("django-insecure-8ovil3xu6=eaoqd#"
+              "-#&ricv159p0pypoh5_lgm*)-dfcjqe=yc")
```

Comment:
The `SECRET_KEY` is hardcoded and should be kept secret. Consider using environment variables to manage this sensitive information.